### ISSUE TITLE
Json buffer overflow

### ISSUE THREAD

**Issue by Invision70** [tdlib/td#397]
Error occurs when receiving messages from different channels. (CGO)
```
-telegram_1         | [ 1][t 0][1541074977.281727791][JsonBuilder.h:746][&jb.string_builder().is_error()] Json buffer overflow
parser-telegram_1         | [ 0][t 0][1541074977.281904697][ClientJson.cpp:41][&!str.empty() && str.back() == '}']
-telegram_1         | SIGABRT: abort
-telegram_1         | PC=0x7fc28e2cce97 m=18 sigcode=18446744073709551610
-telegram_1         | 
-telegram_1         | goroutine 0 [idle]:
-telegram_1         | runtime: unknown pc 0x7fc28e2cce97
-telegram_1         | stack: frame={sp:0x7fc23e7fb8b0, fp:0x0} stack=[0x7fc23dffc1d0,0x7fc23e7fbdd0)
-telegram_1         | 00007fc23e7fb7b0:  0000000000000000  8e241e8c8beab400 
```

**Comment by levlam** [comment]
Current TDLib JSON interface has limited to responses of less than 500 kB. This is fixed in our private version and the restriction will be completely removed in TDLib 1.4.0. For now as a workaround, you can decrease limit to getChatHistory to receive less messages per request. This will not affect performance of message receiving in any way.

**Comment by levlam** [comment]
Fixed in TDLib 1.4.0. The limit is completely removed.
